integr8ly/tutorial-web-app
# feat(fonts): Upgrade to Red Hat fonts

## Motivation
Update the webapp to utilize the Red Hat fonts, replacing Overpass.

## What
- Replace Overpass with Red Hat fonts (RedHatText and RedHatDisplay).
- Update HTML semantics to proper usage and include SkipToContent component for screen readers.

## Why
Move webapp in alignment with other Red Hat products.

## How
Download the latest PatternFly-Core package; add proper wrapping class; include font assets.

## Verification Steps

1. Download PR
2. Run `yarn install && yarn build` to install the latest PatternFly-Core package and rebuild the UI.
3. Run `yarn start:dev` to start up the webapp and test that the new fonts are rendering.
4. Check rendered UI against included screenshots to verify that they match.

## Checklist:


## Progress


## Additional Notes

![Screen Shot 2019-07-17 at 11 09 20 AM](https://user-images.githubusercontent.com/4032718/61387105-7d6d2d80-a883-11e9-84c7-d4e954892a1d.png)
![Screen Shot 2019-07-17 at 11 09 39 AM](https://user-images.githubusercontent.com/4032718/61387106-7d6d2d80-a883-11e9-96ef-0fdc300de493.png)
![Screen Shot 2019-07-17 at 11 09 48 AM](https://user-images.githubusercontent.com/4032718/61387107-7e05c400-a883-11e9-89f6-e8c50f6123a2.png)
![Screen Shot 2019-07-17 at 11 09 58 AM](https://user-images.githubusercontent.com/4032718/61387108-7e05c400-a883-11e9-8618-b6d3551269a8.png)
![Screen Shot 2019-07-17 at 11 10 06 AM](https://user-images.githubusercontent.com/4032718/61387109-7e05c400-a883-11e9-8cef-96691ebed4b5.png)
